src/waypoint.jsx

```diff
@@ -107,6 +120,13 @@ function computeOffsetPixels(offset, contextHeight) {
   }
 }
 
+/**
+ * @param {React.element} Component
+ * @return {bool} Whether the component is a DOM Element
+ */
+function isDomElement(Component) {
+  return (typeof Component.type === 'string');
```

Comment:
> "When an element's type is a string, it represents a DOM node with that tag name" 

from [the docs](https://facebook.github.io/react/blog/2015/12/18/react-components-elements-and-instances.html#dom-elements)

I'll add to the jsdoc